### PR TITLE
MAGENTO2-148

### PR DESCRIPTION
**Pull request by Skistims**
added parent sku to the the internal order info

### CODE REVIEW

File: src/Model/Export/Product.php [hunk 1]
```diff
@@ -268,6 +268,10 @@ public function setInternalOrderInfo()
             'product_id'    => $this->item->getId(),
             'item_type'     => $this->item->getTypeId()
         ];
+        if (is_object($this->parent)) {
```
Comment (augsteyer) [inline src/Model/Export/Product.php]: best if ```instanceof``` is used here. An instance that supports a call to ```getSku()``` and ```getTypeId()``` that is

**Review by augsteyer — CHANGES_REQUESTED** [review]
also need to direct to 'develop' branch

**Comment by andre-kraus** [comment]
Don't forget to update the CHANGELOG.md file as well.

**Comment by Skistims** [comment]
@augsteyer done

**Review by augsteyer — APPROVED** [review]

**Pull request merged**